resources/sass/pages/_torrents.scss

```diff
@@ -118,16 +118,16 @@
 .torrent-search--list__row {
   display: grid;
   grid-template-rows: auto;
-  grid-template-columns: 60px 100px 1fr 150px 60px 100px 50px 50px 50px 120px;
-  grid-template-areas: "poster format overview buttons rating size seeders leechers completed age";
+  grid-template-columns: 60px 100px 1fr 150px 60px 100px 50px 50px 50px 120px 50px;
+  grid-template-areas: "poster format overview buttons rating size seeders leechers completed age action";
```

Comment:
This removes 50px from the overview. Have you checked if it works at the specified resolution?